.env-skit

```diff
@@ -0,0 +1,36 @@
+USE_WEBUI=true
+ALCHEMY_API_KEY=a19a44dd53585567b29bbf17689ac5c23318e3cc
```

Comment:
No. You are right. The .gitignore missed it and I don't remember checking it in. Thanks for catching it.